src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -94,4 +120,60 @@ impl Kbc {
             .ok_or_else(|| anyhow!("TEE rsa key missing"))?;
         response.decrypt_output(key)
     }
+
+    fn tee(&self) -> String {
+        self.tee.clone()
```

Comment:
I suggest the return value here be `&str`, then the caller can decide whether to `clone()` or not. This helps to decrease unnecessary memory copy.